index.bs

```diff
@@ -13202,6 +13394,69 @@ The characteristics of a namespace object are described in [[#namespace-object]]
 </div>
 
 
+<h3 id="es-modules">Modules</h3>
+
+Modules are reified as entries in the [=module map=] as [=Synthetic Module
+Records=], allowing them to be imported as ECMAScript modules.
+
+<div algorithm>
+  To <dfn>create a synthetic module record</dfn> for a [=module=] |module| in a
+  [=Realm=] |realm|, perform the following steps:
+
+    1.  Let |exports| be « ».
+    1.  [=list/For each=] [=module member=] |member| of |module|:
+        1.  Assert: |member| is not annotated with either the [{{Exposed}}]
+                    or [{{SecureContext}}] extended attributes.
+        1.  Append the [=identifier=] of |member| to |exports|.
+    1.  Let |moduleRecord| be ! [=CreateSyntheticModule=](|exports|, the following steps, |realm|, undefined), with the following steps:
+        1. [=list/For each=] [=module member=] |member| of |module|:
+            1.  Let |id| be |member|'s [=identifier=].
+            1.  If |member| is an [=interface=]:
+                1.  Let |interfaceObject| be the result of [=create an interface
+                    object|creating an interface object=] for |member| with |id|
+                    in |realm|.
+                1.  Perform ! [=SetSyntheticModuleExport=](|moduleRecord|, |id|,
+                    |interfaceObject|).
+            1.  Otherwise, if |member| is an [=operation=]:
+                1.  Let |method| be the result of [=creating an operation function=]
+                    given |member|, |module|, and |realm|.
+                1.  Perform ! [=SetSyntheticModuleExport=](|moduleRecord|, |id|,
+                    |method|).
+            1.  Otherwise:
+                1.  Assert: |member| is an [=attribute=].
+                1.  Assert: The module evaluation steps will set each attribute.
+        1.  Run the [=module evaluation steps=] for the [=module/specifier=] of
+            |module|, if any are present.
```

Comment:
I don't think they're "for the specifier of _module_", I think they're just "for _module_", right?